packages/core/src/pagination/register.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@cds/core/button/register.js';
+import '@cds/core/icon/register.js';
+import '@cds/core/input/register.js';
+import { registerElementSafely } from '@cds/core/internal';
+import { ClarityIcons } from '../icon/icon.service.js';
```

Comment:
Import path should be `import { ClarityIcons } from '@cds/core/icon/icon.service.js';`